tests/snapshots/src/main/generated/minimized/Annotations.java

```diff
@@ -33,18 +33,25 @@
 //^^^^^^^^ reference java/lang/annotation/Retention#
 //         ^^^^^^^^^^^^^^^ reference java/lang/annotation/RetentionPolicy#
 //                         ^^^^^^^ reference java/lang/annotation/RetentionPolicy#RUNTIME.
-@Target(value = {CONSTRUCTOR, FIELD, LOCAL_VARIABLE, METHOD, PACKAGE, PARAMETER, TYPE})
+@Target(value = {CONSTRUCTOR,
 //^^^^^ reference java/lang/annotation/Target#
 //      ^^^^^ reference java/lang/annotation/Target#value().
 //               ^^^^^^^^^^^ reference java/lang/annotation/ElementType#CONSTRUCTOR.
-//                            ^^^^^ reference java/lang/annotation/ElementType#FIELD.
-//                                   ^^^^^^^^^^^^^^ reference java/lang/annotation/ElementType#LOCAL_VARIABLE.
-//                                                   ^^^^^^ reference java/lang/annotation/ElementType#METHOD.
-//                                                           ^^^^^^^ reference java/lang/annotation/ElementType#PACKAGE.
-//                                                                    ^^^^^^^^^ reference java/lang/annotation/ElementType#PARAMETER.
-//                                                                               ^^^^ reference java/lang/annotation/ElementType#TYPE.
+        FIELD,
+//      ^^^^^ reference java/lang/annotation/ElementType#FIELD.
+        LOCAL_VARIABLE,
+//      ^^^^^^^^^^^^^^ reference java/lang/annotation/ElementType#LOCAL_VARIABLE.
+        METHOD,
+//      ^^^^^^ reference java/lang/annotation/ElementType#METHOD.
+        PACKAGE,
+//      ^^^^^^^ reference java/lang/annotation/ElementType#PACKAGE.
+        PARAMETER,
+//      ^^^^^^^^^ reference java/lang/annotation/ElementType#PARAMETER.
+        TYPE}
+//      ^^^^ reference java/lang/annotation/ElementType#TYPE.
+)
 public @interface Annotations {
-//                ^^^^^^^^^^^ definition minimized/Annotations# public @interface Annotations
+//                ^^^^^^^^^^^ definition minimized/Annotations# @Documented @Retention(RetentionPolicy.RUNTIME) @Target({CONSTRUCTOR, FIELD, LOCAL_VARIABLE, METHOD, PACKAGE, PARAMETER, TYPE}) public @interface Annotations
```

Comment:
👀 Very nice, the source code uses an explicit `value =` and the signature doesn't. Will be exciting to see once we start using this signature printer against externally defined symbols!